api/v1alpha1/cdapmaster_types.go

```diff
@@ -76,6 +76,8 @@ type CDAPMasterSpec struct {
 	Router RouterSpec `json:"router,omitempty"`
 	// UserInterface is specification for the CDAP UI service.
 	UserInterface UserInterfaceSpec `json:"userInterface,omitempty"`
+	// SupportBundle is specification for the CDAP support-bundle service.
+	SupportBundle SupportBundleSpec `json:"supportBundle,omitempty"`
```

Comment:
Can you make this optional ? so we don't already start up support bundle service